### DISCUSSION TITLE
When is gRPC instrumentation graduating out of alpha?

### DISCUSSION THREAD

**Discussion by ehsannas** [open-telemetry/opentelemetry-java-instrumentation#10825] [Q&A]
Many library developers are often not allowed to / discouraged from adding dependencies on non-stable packages.

As of Mar 12, 2024, the latest `opentelemetry-grpc-1.6` version is [2.1.0-alpha](https://mvnrepository.com/artifact/io.opentelemetry.instrumentation/opentelemetry-grpc-1.6/2.1.0-alpha). We are wondering what contributes to the decision for whether a package is alpha/beta/stable? And when do you think we can expect the gRPC instrumentation to graduate to beta and/or stable?

**Comment by trask** [comment]
hi @ehsannas! the primary blocker is that OpenTelemetry RPC semantic conventions are still declared experimental: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-spans.md

**Comment by ehsannas** [comment]
Hey @trask ! Thanks for the quick response. I see. I'll be following the progress of RPC semantic conventions then. Thanks!
